datasource/ethereum/src/block_ingestor.rs

```diff
@@ -97,17 +97,27 @@ where
                                 let latest_number = latest_block.number.unwrap().as_u64() as i64;
                                 let head_number = head_block_ptr.number as i64;
                                 let distance = latest_number - head_number;
+                                if distance > 0 {
+                                    info!(
+                                        self.logger,
```

Comment:
Can we combine this with the two messages below? It seems unnecessary to log two messages each time. And let's use more human-readable names for the different values.

`local_block_ptr` -> `current_head_block`
`network_block_ptr` -> `latest_head_block`
`distance` -> `blocks_behind`?